### PR TITLE
Petstore 2019

### PR DESCRIPTION
**Pull request by hannah-bulmer**
Have updated the Petstore component to be compliant with our 2019 coding standards and styles, removed outdated libraries, etc

Fixes following issues:
https://github.com/elasticio/petstore-component-nodejs/issues/12: eslinting
https://github.com/elasticio/petstore-component-nodejs/issues/8: adds enums to metatdata
https://github.com/elasticio/petstore-component-nodejs/issues/7: adds descriptions to actions/triggers

I'm not sure if it's worth noting/doing anything about since this is just a tool for teaching, but the POST /pet endpoint does not actually post anything anywhere, so this action does not actually do anything at all...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,10 +16,17 @@
         "npm": ">=5.4"
     },
     "dependencies": {
-        "elasticio-sailor-nodejs": "2.2.1",
+        "@elastic.io/component-commons-library": "0.0.6",
+        "chai": "^4.2.0",
```
Comment (dvd-z) [inline package.json]: carat

File: verifyCredentials.js [hunk 2]
```diff
@@ -1,34 +1,30 @@
...
+      url: '/user/me',
+      method: 'GET',
+    });
 
     // if the request succeeds, we can assume the api key is valid
-    return request.get(requestOptions);
-}
\ No newline at end of file
+    return true;
+  } catch (e) {
```
Comment (dvd-z) [inline verifyCredentials.js]: unused var

Comment (hannah-bulmer) [reply verifyCredentials.js]: eslint thinks its useful 😢

File: component.json [hunk 3]
```diff
@@ -9,36 +9,34 @@
         "label": "API key",
         "required": true,
         "viewClass": "TextFieldWithNoteView",
-        "note": "Please use <b>elasticio</b> as API key. For more details see <a href='https://petstore.elastic.io/docs/' target='_blank'>Petstore API docs</a>."
+        "note": "Please use <b>secret</b> as API key. For more details see <a href='https://petstore.elastic.io/docs/' target='_blank'>Petstore API docs</a>."
```
Comment (jhorbulyk) [inline component.json]: I think this value is referring to the value that is expected by the server.

**Pull request merged**